### PR TITLE
Feature/orb openapi policy

### LINKED ISSUE #212: Policy Openapi

### PR DESCRIPTION
**Pull request by lterranova**
Resolves #212

### CODE REVIEW

File: policies/api/openapi.yaml [hunk 1]
```diff
@@ -6,67 +6,760 @@ servers:
...
+          description: Timestamp of creation
+        version:
+          type: integer
+          readOnly: true
+          example: 0
+          description: A monotonically increasing counter starting at 0 on creation and increasing with each policy update.
+        policy:
+          type: string
+          example:
+            value: "version: \"1.0\"\nvisor:\n    foo: \"bar\""
```
Comment (dscabral) [inline policies/api/openapi.yaml:614]: Only policy_data has a yaml format, policy it's a json, you should change do json format

File: policies/api/openapi.yaml [hunk 2]
```diff
@@ -6,67 +6,760 @@ servers:
...
+          example: my-dataset
+        agent_group_id:
+          type: string
+          format: uuid
+          description: A unique identifier of an agent_group
+        agent_policy_id:
+          type: string
+          format: uuid
+          description: A unique identifier of an agent_policy
+        sink_id:
```
Comment (dscabral) [inline policies/api/openapi.yaml:689]: sink_id should be an array of strings

File: policies/api/openapi.yaml [hunk 3]
```diff
@@ -6,67 +6,760 @@ servers:
...
+          example: my-dataset
+        agent_group_id:
+          type: string
+          format: uuid
+          description: Unique identifier (UUID) of an agent_group
+        agent_policy_id:
+          type: string
+          format: uuid
+          description: Unique identifier (UUID) of an agent_policy
+        sink_id:
```
Comment (dscabral) [inline policies/api/openapi.yaml:744]: Should be an array of strings

File: policies/api/openapi.yaml [hunk 4]
```diff
@@ -6,67 +6,760 @@ servers:
...
+            items:
+              $ref: '#/components/schemas/ConfigEntrySchema'
+    DatasetUpdateReqSchema:
+      type: object
+      properties:
+        name:
+          type: string
+          description: A unique name label
+          example: my-dataset
+        sink_id:
```
Comment (dscabral) [inline policies/api/openapi.yaml:657]: Should be an array of strings

**Review by dscabral — CHANGES_REQUESTED** [review]

**Review by dscabral — APPROVED** [review]

**Review by weyrick — APPROVED** [review]
Ok I think this is ok to land now, but be aware that in the future we may accept a policy format other than `yaml` so we will have to update the docs at that point.

**Pull request merged**